test/controllers/smart_proxies_controller_test.rb

```diff
@@ -196,6 +200,7 @@ def test_refresh_fail
 
   test '#expire_logs' do
     proxy = smart_proxies(:logs)
+    proxy.stubs(:associate_features)
```

Comment:
Why do we need this when we have the global `setup do`?

Reply:
Oh a leftover, thanks.